app/forms/idv/inherited_proofing/va/form.rb

```diff
@@ -4,16 +4,37 @@ module Va
       class Form < Idv::InheritedProofing::BaseForm
         class << self
           def required_fields
-            @required_fields ||= %i[first_name last_name birth_date ssn address_street address_zip]
+            @required_fields ||= %i[first_name
+                                    last_name
+                                    birth_date
+                                    ssn
+                                    address_street
+                                    address_zip]
           end
 
           def optional_fields
-            @optional_fields ||= %i[phone address_street2 address_city address_state
-                                    address_country]
+            @optional_fields ||= %i[phone
+                                    address_street2
+                                    address_city
+                                    address_state
+                                    address_country
+                                    service_error]
```

Comment:
I see this was already done as a class method, but IMO these are a better fit for a constant like `OPTIONAL_FIELDS`, I'd recommend switching in a follow-up PR